voteswap/templates/base.html

```diff
@@ -47,9 +49,14 @@
       </div>
     </footer>
 
-    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
-    <script src="{% static 'jquery.slim.min.js' %}"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
```

Comment:
Its a minimal version of jq that doesn't include Ajax. I can move the version i need into the vendor_static directory if you dont want to load from a cdn (but I would think the cdn would be faster)